website/ref/language.md

```diff
@@ -949,19 +962,24 @@ pattern: `put */*.cpp[nomatch-ok]` unfortunately does not do the correct thing.
         `upper`. See the Is\* functions [here](https://godoc.org/unicode) for
         their definitions.
 
-    Note the following caveats:
+Note the following caveats:
+
+-   Local matchers chained together in separate modifiers are OR'ed. For
+    instance, `?[set:aeoiu][digit]` matches all files with the chars `aeoiu` or
+    containing a digit.
 
-    -   Multiple matchers, they are OR'ed. For instance, ?[set:aeoiu][digit]
-        matches `aeoiu` and digits.
+-   Local matchers combined in the same modifier, such as `?[set:aeoiu digit]`,
+    are OR'ed but in a manner something like a SQL outer join. This results in
```

Comment:
This behavior is more easily and unambiguously explained by saying that `?[set:aeoiu digit]` expands to `?[set:aeiou] ?[digit]`.

Reply:
Your example does not match the behavior I'm seeing. Either standalone or bracketed by `*` wildcards. For example:

```
 macpro  ~/tmp  > echo ?[set:aeoiu digit]
1 a 1 a
 macpro  ~/tmp  > echo ?[set:aeoiu] ?[digit]
a 1
 macpro  ~/tmp  > echo *?[set:aeoiu digit]*
1 :3 a a1 b2 d:4 1 :3 a a1 b2 d:4
 macpro  ~/tmp  > echo *?[set:aeoiu]?[digit]*
a1
```

Frankly, I think combining local modifiers in this fashion should not be allowed. At least not until it has a behavior that can be explained to an average shell user. So I'm simply going to modify the comment to say "do not use".